lib/hmmlearn/_hmmc.pyx

```diff
@@ -1,5 +1,4 @@
 # cython: language_level=3, boundscheck=False, wraparound=False
-
```

Comment:
please undo all these blank line changes.

Reply:
I've done my best to catch as many of these as possible.